objectherkenning_openbare_ruimte/performance_evaluation_pipeline/metrics/per_pixel_stats.py

```diff
@@ -57,7 +57,12 @@ def get_statistics(self, precision: int = 3) -> Dict[str, float]:
 
         Parameters
         ----------
-        precision: int = 3
+        size_all: bool
+            Indicates whether the statistics are computed over all bounding box
+            sizes (True) or whether a subset is evaluated (False). If False,
+            only True Positive, True Negative, and Precision are returned as the
+            other statistics are not meaningful.
```

Comment:
Here you say that if size_all = False, only TP, TN and Precision are returned, but then in the return statement you return TP, FN and Recall.